### PR TITLE
[full-ci] Keyboard navigation disable for inputs

### PR DESCRIPTION
**Pull request by lookacat**
## Description
Disable keyboard navigation on textarea and input

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/src/components/FilesList/KeyboardActions.vue [hunk 1]
```diff
@@ -86,6 +86,12 @@ export default defineComponent({
     }),
 
     areCustomKeyBindingsEnabled() {
+      const activeElementTag = document.activeElement.tagName
+      if (
+        activeElementTag.toLowerCase() === 'textarea' ||
+        activeElementTag.toLowerCase() === 'input'
```
Comment (kulmann) [inline packages/web-app-files/src/components/FilesList/KeyboardActions.vue:92]: Please also add `select`.

Comment (kulmann) [reply packages/web-app-files/src/components/FilesList/KeyboardActions.vue:92]: Shorter version of this code would be:
```js
['textarea', 'input', 'select'].includes(activeElementTag.toLowerCase())
```

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/28384/54/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAcceptShares-acceptShares_feature-L228.png
  


</p></details>
:boom: The oCISSharingBasic tests pipeline failed. The build has been cancelled.

File: packages/web-app-files/src/components/FilesList/KeyboardActions.vue [hunk 2]
```diff
@@ -86,6 +86,8 @@ export default defineComponent({
     }),
 
     areCustomKeyBindingsEnabled() {
+      const activeElementTag = document.activeElement.tagName
+      if (['textarea', 'input', 'select'].includes(activeElementTag.toLowerCase())) return true
```
Comment (JanAckermann) [inline packages/web-app-files/src/components/FilesList/KeyboardActions.vue:90]: Personally not a fan of the inline return without  {}

**Review by JanAckermann — APPROVED** [review]
LGTM, one objection, but personal style I guess

File: packages/web-app-files/src/components/FilesList/KeyboardActions.vue (hunk 2 above)
Comment (kulmann) [reply packages/web-app-files/src/components/FilesList/KeyboardActions.vue:90]: Same. Code style in web and related repos is to not inline return statements.

Comment (JanAckermann) [reply packages/web-app-files/src/components/FilesList/KeyboardActions.vue:90]: Tried to find a eslint rule set plugin, but nothing found unfortantly

**Pull request merged**